src/patternfly/components/Menu/menu.scss

```diff
@@ -0,0 +1,294 @@
+  --pf-c-menu__group-title--PaddingRight: var(--pf-c-menu__item--PaddingRight);
+  --pf-c-menu__group-title--PaddingBottom: var(--pf-c-menu__item--PaddingBottom);
+  --pf-c-menu__group-title--PaddingLeft: var(--pf-c-menu__item--PaddingLeft);
+  --pf-c-menu__group-title--FontSize: var(--pf-global--FontSize--sm);
+  --pf-c-menu__group-title--FontWeight: var(--pf-global--FontWeight--semi-bold);
+  --pf-c-menu__group-title--Color: var(--pf-global--Color--dark-200);
+
+  // description
+  --pf-c-menu__item-description--FontSize: var(--pf-global--FontSize--xs);
+  --pf-c-menu__item-description--Color: var(--pf-global--Color--200);
+
+  // icon
+  --pf-c-menu__item-icon--MarginRight: var(--pf-global--spacer--sm);
+
+  // toggle icon
+  --pf-c-menu__item-toggle-icon--PaddingRight: var(--pf-global--spacer--sm);
+  --pf-c-menu__item-toggle-icon--PaddingLeft: var(--pf-global--spacer--sm);
+
+  // select icon
+  --pf-c-menu__item-select-icon--MarginLeft: var(--pf-global--spacer--sm);
+  --pf-c-menu__item-select-icon--Color: var(--pf-global--active-color--100);
+  --pf-c-menu__item-select-icon--FontSize: var(--pf-global--icon--FontSize--sm);
+
+  // external icon
+  --pf-c-menu__item-external-icon--MarginLeft: var(--pf-global--spacer--sm);
+  --pf-c-menu__item-external-icon--Color: var(--pf-global--link--Color);
+  --pf-c-menu__item-external-icon--FontSize: var(--pf-global--icon--FontSize--sm);
+
+  // action
+  --pf-c-menu__item-action--PaddingTop: var(--pf-global--spacer--sm);
+  --pf-c-menu__item-action--PaddingRight: var(--pf-global--spacer--md);
+  --pf-c-menu__item-action--PaddingBottom: var(--pf-global--spacer--sm);
+  --pf-c-menu__item-action--PaddingLeft: var(--pf-global--spacer--md);
+  --pf-c-menu__item-action-icon--Color: var(--pf-global--Color--dark-200);
+  --pf-c-menu__item-action-icon--Height: calc(var(--pf-c-menu__item--FontSize) * var(--pf-c-menu__item--LineHeight));
+  --pf-c-menu__item-action--hover__icon--Color: var(--pf-global--Color--dark-100);
+  --pf-c-menu__item-action--m-favorite__icon--Color: var(--pf-global--disabled-color--200);
+  --pf-c-menu__item-action--m-favorite__icon--FontSize: var(--pf-global--icon--FontSize--sm);
+  --pf-c-menu__item-action--m-favorite--m-favorited__icon--Color: var(--pf-global--palette--gold-400);
+
+  padding-top: var(--pf-c-menu--PaddingTop);
+  padding-bottom: var(--pf-c-menu--PaddingBottom);
+  background-color: var(--pf-c-menu--BackgroundColor);
+  box-shadow: var(--pf-c-menu--BoxShadow);
+
+  &.pf-m-flyout {
+    .pf-c-menu__sublist {
+      position: absolute;
+      top: var(--pf-c-menu--m-flyout__sublist--Top);
+      left: 100%;
+    }
+
+    .pf-c-menu__list-item {
+      &:hover,
+      &:focus-within {
+        color: unset;
+        background-color: unset;
+      }
+    }
+
+    .pf-c-menu__item {
+      &:hover,
+      &:focus-within {
+        color: var(--pf-c-menu__list-item--hover--Color);
+        background-color: var(--pf-c-menu__list-item--hover--BackgroundColor);
+      }
```

Comment:
why do you need to move the `:hover/focus-within` styles in a flyout menu? Assuming it's necessary, you probably need to scope the classes, or maybe add a class to a list-item that has a submenu and scope it to that. Here's what it looks like if you nest a list with actions in the flyout menu now

<img width="594" alt="Screen Shot 2020-08-19 at 6 22 17 PM" src="https://user-images.githubusercontent.com/35148959/90699524-1d487800-e249-11ea-9437-b132fc01c000.png">